dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/AssignTrn/NoTrn.cshtml

```diff
@@ -0,0 +1,24 @@
+@page "/admin/users/{userId}/no-trn"
+@model TeacherIdentity.AuthServer.Pages.Admin.AssignTrn.NoTrn
+@{
+    ViewBag.Title = @Html.DisplayNameFor(m => m.HasNoTrn);
+}
+
+@section BeforeContent {
+    <govuk-back-link asp-page="/Admin/User" asp-route-userId="@Model.UserId" />
```

Comment:
Not sure if makes more sense to redirect back to /Admin/Users here

Reply:
Agreed. We can loop back and make sure we have the right links from both the users list and user detail page later on.